model/waterfall.go

```diff
@@ -224,3 +228,37 @@ func GetWaterfallBuildVariants(ctx context.Context, versionIds []string) ([]Wate
 
 	return res, nil
 }
+
+// GetNextActiveWaterfallVersion returns the next active version on the waterfall.
```

Comment:
Could you be more specific with "next" here? Looking at the code I can tell you mean newer, but next could also be interpreted as older, as in next when looking at the waterfall.